containers/Ideas/Idea.js

```diff
@@ -180,94 +181,65 @@ export const Idea = (props) => {
     }
   }
 
-  const handleCommentSubmit = (values) => {
+  const handleComment = (values) => {
     commentInputRef.current.blur()
     AsyncStorage.getItem('authToken')
       .then((token) => {
         values.agreed_terms_of_use = true
-        return API.addComment(
-          contentObjectOfComment.contentType,
-          contentObjectOfComment.pk,
-          values,
-          token
-        )
+        const payload = getCommentPayload(values, token)
+
+        return isEditing
+          ? API.editComment(...payload)
+          : API.addComment(...payload)
```

Comment:
this was a bigger refactor actually, we used two functions that did almost the same and it was quite hard to add or change things, because you had to do it for both. I think now it is better separated and easier to maintain. But sure, i can revert it if the majority prefers the other way.

Reply:
it's more about the payload, I just find it easier to see what's happening when it's in the submit but also happy to go with majority

Reply:
@phillimorland do you have a concrete suggestion how i could make it more readable here?

Reply:
It's ok like this for me, I personally would prefer a simple `if` as I'm more used to that:
```
if (isEditing) return API.editComment(...payload)
return API.addComment(...payload)

Reply:
I agree that we should use `if` over ternary which are less readable, I thought we decided to only use ternary in simple cases, like where everything can be on 1 line?!

Reply:
yes true, okay i will get rid of the ternary notation also in `getCommentPayload`

Reply:
Sorry for this late comment, but tbh I find the above solution more readable and nicer structured.. :see_no_evil: Totally personal preferences, maybe before next app sprint, we can talk about that again.